### ISSUE TITLE
External meeting window and not closed after terminating

### ISSUE THREAD

**Issue by a7mdragab1** [jitsi/jitsi-meet-flutter-sdk#92]
Dears,
I have aproblem with the package!

when upgrading to any version more than
jitsi_meet_flutter_sdk: 0.4.2

The meeting is opened in a separate screen, but when the meeting ends, the screen is not removed from the background. If clicked, it reopens the meeting.

Look at this scenario
![image](https://github.com/user-attachments/assets/94818d5e-38fa-4254-a7f4-6343b1493b20)

So, I am stuck at this version.

This is the basic example in docs
```
var jitsiMeet = JitsiMeet();
var options = JitsiMeetConferenceOptions(
  serverURL: "https://meet.jit.si",
  room: "jitsiIsAwesomeWithFlutter",
  configOverrides: {
    "startWithAudioMuted": false,
    "startWithVideoMuted": false,
    "subject": "Jitsi with Flutter",
  },
  featureFlags: {"unsaferoomwarning.enabled": false},
  userInfo: JitsiMeetUserInfo(displayName: "Flutter user", email: "user@example.com"),
);
jitsiMeet.join(options);
```
Even I tried serverURL with my url, both have the same behavior

**Comment by Calinteodor** [comment]
I managed to test using the below configuration and an Android 14 device

![image](https://github.com/user-attachments/assets/46172c99-21cd-40a8-9d2c-4f56f1b5f01c)

and the sample app right here https://github.com/jitsi/jitsi-meet-sdk-samples/tree/master/flutter.

I customised prosody to accept the end conference feature and I am unable to encounter what you are saying.

When I end meeting for all, all participants are thrown out of the meeting view, and I, the moderator, from flutter, get an OK dialog and when I press it I navigate back to the main view. No other view remain open in the background. 

Maybe try to cover the steps that I mentioned above and see if you get the same results with your customised server.

**Comment by a7mdragab1** [comment]
> I managed to test using the below configuration and an Android 14 device
> 
> ![image](https://private-user-images.githubusercontent.com/26413496/377069607-822fe4f4-6ab3-4ec5-88b7-92ba5d437f06.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3Mjk0MDU5NTIsIm5iZiI6MTcyOTQwNTY1MiwicGF0aCI6Ii8yNjQxMzQ5Ni8zNzcwNjk2MDctODIyZmU0ZjQtNmFiMy00ZWM1LTg4YjctOTJiYTVkNDM3ZjA2LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDEwMjAlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQxMDIwVDA2MjczMlomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWE4N2M4ZWRmZmEyYzEzYzcwZTA5OGMzYTlhZGE3ZWE1MmVlZjk4YzhmOGE5N2FmNTk1ODE0OTA2NDJjZGZiYTEmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.1X7ktlKpfSrFXuI_tKIoLYqPtFctncRTCufk4a892mw)
> 
> and the sample app right here https://github.com/jitsi/jitsi-meet-sdk-samples/tree/master/flutter.
> 
> I customised prosody to accept the end conference feature and I am unable to encounter what you are saying.
> 
> When I end meeting for all, all participants are thrown out of the meeting view, and I, the moderator, from flutter, get an OK dialog and when I press it I navigate back to the main view. No other views remain open in the background.
> 
> Maybe try to cover the steps that I mentioned above and see if you get the same result with your customised server.

Thanks for your reply
Can you upgrade to latest flutter and try creating a new project and copy that code to it and try

**Comment by saghul** [comment]
Quick question: does the screen disappear when you tap on the ok button in the dialog?

**Comment by a7mdragab1** [comment]
> Quick question: does the screen disappear when you tap on the ok button in the dialog?

No, this exactly what I am doing to reproduce the image in the question
It returns to the main app but keeps another window opened for the meeting

**Comment by a7mdragab1** [comment]
> I customised prosody to accept the end conference feature and I am unable to encounter what you are saying.

What is meant by that?

**Comment by Calinteodor** [comment]
> > I managed to test using the below configuration and an Android 14 device
> > ![image](https://private-user-images.githubusercontent.com/26413496/377069607-822fe4f4-6ab3-4ec5-88b7-92ba5d437f06.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3Mjk0MDU5NTIsIm5iZiI6MTcyOTQwNTY1MiwicGF0aCI6Ii8yNjQxMzQ5Ni8zNzcwNjk2MDctODIyZmU0ZjQtNmFiMy00ZWM1LTg4YjctOTJiYTVkNDM3ZjA2LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDEwMjAlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQxMDIwVDA2MjczMlomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWE4N2M4ZWRmZmEyYzEzYzcwZTA5OGMzYTlhZGE3ZWE1MmVlZjk4YzhmOGE5N2FmNTk1ODE0OTA2NDJjZGZiYTEmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.1X7ktlKpfSrFXuI_tKIoLYqPtFctncRTCufk4a892mw)
> > and the sample app right here https://github.com/jitsi/jitsi-meet-sdk-samples/tree/master/flutter.
> > I customised prosody to accept the end conference feature and I am unable to encounter what you are saying.
> > When I end meeting for all, all participants are thrown out of the meeting view, and I, the moderator, from flutter, get an OK dialog and when I press it I navigate back to the main view. No other views remain open in the background.
> > Maybe try to cover the steps that I mentioned above and see if you get the same result with your customised server.
> 
> Thanks for your reply Can you upgrade to latest flutter and try creating a new project and copy that code to it and try
> 
> ```
> flutter doctor
> Doctor summary (to see all details, run flutter doctor -v):
> [√] Flutter (Channel beta, 3.26.0-0.1.pre, on Microsoft Windows [Version 10.0.22631.4317], locale en-US)
> [√] Windows Version (Installed version of Windows is version 10 or higher)
> [√] Android toolchain - develop for Android devices (Android SDK version 35.0.0)
> [√] Chrome - develop for the web
> [√] Visual Studio - develop Windows apps (Visual Studio Community 2022 17.10.4)
> [√] Android Studio (version 2024.2)
> [√] VS Code (version 1.94.2)
> [√] Connected device (5 available)
> [√] Network resources
> 
> • No issues found!
> ```

I am alreay using latest flutter version
![image](https://github.com/user-attachments/assets/9b559b30-0f8e-4c2e-8b43-d1c91e01ae3c)

**Comment by bilal-codingkey** [comment]
> > Quick question: does the screen disappear when you tap on the ok button in the dialog?
> 
> No, this exactly what I am doing to reproduce the image in the question It returns to the main app but keeps another window opened for the meeting

@saghul It does happen, that when a user hung up a meeting, it's redirected to main app but the flutter meeting instance is still available which user can simply select to rejoin the meeting as illustrated in image 3 of this thread's first post. 
My use case requires to prevent user from joining once they have left the meeting.

**Comment by saghul** [comment]
I think what you are seeing is the recents list entry. This should fix it: https://github.com/jitsi/jitsi-meet-flutter-sdk/pull/115

**Comment by bilal-codingkey** [comment]
@saghul v11.1.1, does indeed fix this, I think we can close it.

**Comment by saghul** [comment]
Thanks for confirming!
